int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -492,6 +508,9 @@ function setEventsForListeners(checkoutStage) {
 	setEdqInputSelectors(checkoutStage);
 	removeMultipleEDQSuggestion();
 	edqSetGlobalIntuitiveConfiguration();
+	addEventOnElement("[name=" + edqAddressLine1Id.name + "]", "keypress", setCountryField);
```

Comment:
Why are these events added?

Reply:
These events are been added to the address line input to do the check on the address country options in case this one is set with a blank option (by default the store has a blank option); Global Intuitive has a returning country label (component["countryISO1"]); but these value returns an ISO3 country code and the default code for the store is working on the ISO2 country code; so I place these events to handle the address input when you use Global Intuitive.